crates/hc_sandbox/examples/setup_5.rs

```diff
@@ -40,7 +40,9 @@ async fn main() -> anyhow::Result<()> {
             None,
             None,
             false,
+            #[cfg(feature = "unstable-dpki")]
             false,
+            #[cfg(feature = "unstable-dpki")]
             None,
```

Comment:
That wouldn't change much on every occurrence of the generate function, because you need to differentiate between `#[cfg(not(feature="unstable-dpki"))] generate` and `#[cfg(feature="unstable-dpki")] generate_with_dpki`. The `cfg`s on the function declaration can be dropped.